server/core/src/main/scala/sttp/tapir/server/interceptor/exception/ExceptionHandler.scala

```diff
@@ -1,19 +1,20 @@
 package sttp.tapir.server.interceptor.exception
 
 import sttp.model.StatusCode
-import sttp.tapir.{server, _}
+import sttp.monad.MonadError
+import sttp.tapir._
 import sttp.tapir.server.model.ValuedEndpointOutput
 
-trait ExceptionHandler {
-  def apply(ctx: ExceptionContext): Option[ValuedEndpointOutput[_]]
+trait ExceptionHandler[F[_]] {
```

Comment:
maybe we can add a method to conveniently create pure handlers, that is given `ExceptionContext => Option[VEO[_]]`

Reply:
I like the idea but I struggle to create descriptive name for it. I think we can mimic cats effect convention and rename apply to `handleError` and name second method `handleErrorWith`. What do you think?

Reply:
maybe simply `object ExceptionHandler { def pure(...) ... }` ?

Reply:
Yes, I've finally decided to use `ExceptionHandler` comainion object, already in progress :)